lib/mb_getResponse.m

```diff
@@ -21,10 +24,43 @@
         % check if tap and save time (it counts as tap if
         % reponse buttons were released initially)
         if ~istap && any(keyCode)
-            % tap onset time is saved wrt sequence start time
-            currTapOnsets = [currTapOnsets,tapOnset-currSeqStartTime];
+            % tap onset relative to sequence start time
+            tapTime = tapTime-currSeqStartTime; 
+            
+            % append to the vector of tap times
+            tapOnsets = [tapOnsets,tapTime];
+            
+            % set flag
             istap = true;
+            
+            % ----- log tsv -----
+            % now we have some time before the subject taps again so let's
+            % write to the log file            
+            % each tap on one row (colum names: subjectNum, runNum, seqi, tapOnset)
+            fprintf(expParam.fidTap, '%d\t%d\t%d\t%f\n', ...
+                                    expParam.subjectNb, ...
+                                    expParam.runNb, ...
+                                    seqi, ...
+                                    tapTime);
+            % ------ bids -------
+            % Find latest pattern for which current tap time is larger 
+            % than it's onset. 
+            currPatIdx = max( find(tapTime > currSeq.patternOnset) ); 
```

Comment:
Since I changed the structure array, it gave an error saying too many input arguments. 
I cannot find something smarter than below:
```
for i =1:length(currSeq)
                patternOnset(i) = currSeq(i).onset;
            end
            

            % ------ bids -------
            % Find latest pattern for which current tap time is larger 
            % than it's onset. 
            currPatIdx = max( find(tapTime > patternOnset) ); 
```

Reply:
I think you can do [currSeq.onset]

Reply:
YES! Shall I do it or you are on it?

Reply:
I'll do it